src/user-icon/user-icon.scss

```diff
@@ -64,42 +52,39 @@
     display: block;
     position: absolute;
     object-fit: cover;
-    z-index: 1;
     left: 0;
     top: 0;
-    width: $size;
-    height: $size;
+    width: 100%;
+    height: 100%;
     border-width: 0;
     border-radius: 50%;
   }
 
-  > .iui-user-icons-status {
+  > .iui-status {
     display: block;
     position: absolute;
     right: -2px;
     bottom: -2px;
-    width: $size / 4;
-    height: $size / 4;
+    width: calc(100% / 4);
+    height: calc(100% / 4);
     border-radius: 50%;
-    z-index: 3;
-
     @include themed {
       border: 2px solid t(iui-color-background-1);
       background-color: t(iui-color-background-1);
       box-shadow: inset 0 0 0 1px
         rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-3));
     }
 
-    > .iui-user-icons-status-symbol {
+    > .iui-status-symbol {
```

Comment:
Maybe we could have just iui-status to take care of circular form and element inside.